### PR TITLE
Adding Consul 1.5.3

### PR DESCRIPTION
**Pull request by tryan225**

### CODE REVIEW

File: library/consul [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: c4d239035589dc12fe82fc8550766faf7ec4961d
 Directory: 0.X
 
-Tags: 1.5.2, 1.5, latest
+Tags: 1.5.2, 1.5
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: 53c426f4c5d1cc9517c93de97bf32d56a4d5380d
```
Comment (rboyer) [inline library/consul]:
```suggestion
GitCommit: 9d0aaa3caf747a7022969bcaaff3399fe34efc9c
```

File: library/consul [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+Directory: 0.X
+
 Tags: 1.6.0-beta1
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: c4d239035589dc12fe82fc8550766faf7ec4961d
 Directory: 0.X
 
-Tags: 1.5.2, 1.5, latest
+Tags: 1.5.2, 1.5
```
Comment (rboyer) [inline library/consul]:
```suggestion
Tags: 1.5.3, 1.5, latest
```

File: library/consul [hunk 3]
```diff
@@ -1,12 +1,18 @@
 Maintainers: Consul Team <consul@hashicorp.com> (@hashicorp/consul)
 
+Tags: 1.5.3, 1.5, latest
```
Comment (rboyer) [inline library/consul]:
```suggestion
```

File: library/consul [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+Architectures: amd64, arm32v6, arm64v8, i386
```
Comment (rboyer) [inline library/consul]:
```suggestion
```

File: library/consul [hunk 5]
```diff
@@ -1,12 +1,18 @@
...
+GitRepo: https://github.com/hashicorp/docker-consul.git
```
Comment (rboyer) [inline library/consul]:
```suggestion
```

File: library/consul [hunk 6]
```diff
@@ -1,12 +1,18 @@
...
+GitCommit: 9d0aaa3caf747a7022969bcaaff3399fe34efc9c
```
Comment (rboyer) [inline library/consul]:
```suggestion
```

File: library/consul [hunk 7]
```diff
@@ -1,12 +1,18 @@
...
+Directory: 0.X
```
Comment (rboyer) [inline library/consul]:
```suggestion
```

File: library/consul [hunk 8]
```diff
@@ -1,12 +1,18 @@
...
+
```
Comment (rboyer) [inline library/consul]:
```suggestion
```

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index ef1ec2d..eab60db 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -11,6 +11,6 @@ consul:1.3.1
 consul:1.4
 consul:1.4.5
 consul:1.5
-consul:1.5.2
+consul:1.5.3
 consul:1.6.0-beta1
 consul:latest
diff --git a/consul_latest/Dockerfile b/consul_latest/Dockerfile
index 60879dc..69ee232 100644
--- a/consul_latest/Dockerfile
+++ b/consul_latest/Dockerfile
@@ -2,7 +2,7 @@ FROM alpine:3.9
 MAINTAINER Consul Team <consul@hashicorp.com>
 
 # This is the release of Consul to pull in.
-ENV CONSUL_VERSION=1.5.2
+ENV CONSUL_VERSION=1.5.3
 
 # This is the location of the releases.
 ENV HASHICORP_RELEASES=https://releases.hashicorp.com
```

</details>

**Comment by yosifkit** [comment]
Build test of #6359; e8d2e38b; `amd64` (`consul`):

```console

$ bashbrew build consul:1.6.0-beta1
Using bashbrew/cache:37c8b0a9881eaf1534d115bc787e4a657e38d396bd55b0b8b6c2e57875bd50ad (consul:1.6.0-beta1)
Tagging consul:1.6.0-beta1

$ test/run.sh consul:1.6.0-beta1
testing consul:1.6.0-beta1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build consul:1.5.3
Building bashbrew/cache:3be0dc0bde6827145f0b03437f205f26a9435eb5188354b11ac5f30bf9e5379d (consul:1.5.3)
Tagging consul:1.5.3
Tagging consul:1.5
Tagging consul:latest

$ test/run.sh consul:1.5.3
testing consul:1.5.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build consul:1.4.5
Using bashbrew/cache:ddee8dd3e0be6f60b67141aa5bc2c5ee13a1a9e294b4aa45ab4d0dd9f6bb8074 (consul:1.4.5)
Tagging consul:1.4.5
Tagging consul:1.4

$ test/run.sh consul:1.4.5
testing consul:1.4.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build consul:1.3.1
Using bashbrew/cache:48f844c2d82b7d96272435359d82226637f427b002c4a0d4848b6958b1ea127a (consul:1.3.1)
Tagging consul:1.3.1
Tagging consul:1.3

$ test/run.sh consul:1.3.1
testing consul:1.3.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build consul:1.2.4
Using bashbrew/cache:bf73905287317be7adc56a2ccb56a2c5a5655671f38677e3a2ad9bfa71e28d9c (consul:1.2.4)
Tagging consul:1.2.4
Tagging consul:1.2

$ test/run.sh consul:1.2.4
testing consul:1.2.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build consul:1.1.1
Using bashbrew/cache:19afd997b82332598c1eca97deef2d038add3be7c7e07d591fac46a2eff4935b (consul:1.1.1)
Tagging consul:1.1.1
Tagging consul:1.1

$ test/run.sh consul:1.1.1
testing consul:1.1.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build consul:1.0.8
Using bashbrew/cache:0e469fa5357bae0553361a4583ff9fe3f6567cfa9f16fbbf2bc179621ad1c698 (consul:1.0.8)
Tagging consul:1.0.8
Tagging consul:1.0

$ test/run.sh consul:1.0.8
testing consul:1.0.8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build consul:0.9.4
Using bashbrew/cache:64c0a7e0bfe9ca508dd98c40b12a14e8aa34e3634c7941670f4edbe0a9d0fb0a (consul:0.9.4)
Tagging consul:0.9.4
Tagging consul:0.9

$ test/run.sh consul:0.9.4
testing consul:0.9.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**